scripts/export-analytics.js

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env node
+
+/**
+ * Export eventAggregates data to Biquery
+ */
+
+const {
+	createDatasetIfNotExists,
+	createTableIfNotExists,
+	getTableClient,
+	DATASET,
+	PROJECT_ID
+} = require('./index')
+const db = require('../db')
+const logger = require('../services/logger')('evAggr')
+const { bigQueryTables, collections } = require('../services/constants')
+
+const schema = [
+	{ name: 'channelId', type: 'STRING', mode: 'REQUIRED' },
+	{ name: 'created', type: 'TIMESTAMP', mode: 'REQUIRED' },
+	{ name: 'event_type', type: 'STRING', mode: 'REQUIRED' },
+	{ name: 'earner', type: 'STRING', mode: 'REQUIRED' },
+	{ name: 'count', type: 'STRING', mode: 'REQUIRED' },
+	{ name: 'payout', type: 'STRING', mode: 'REQUIRED' }
```

Comment:
Just talked to @GTitova about this and we need `count` and `payout` to be `NUMERIC`. Can we do that? I am not sure how big the payout number can get here so be careful.

Reply:
It should be ok @samparsky, as we don't need absolute precision in the analytics

Reply:
Yeah I have made the change